src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -427,7 +440,24 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
       if (this.startupPeriod) {
         targetBitrate = VideoPriorityBasedPolicy.DEFAULT_BANDWIDTH_KBPS;
       } else {
-        targetBitrate = this.downlinkStats.bandwidthEstimateKbps;
+        // We rely on our target bitrate being above what we are receiving to mark a probe as complete,
+        // however in browsers, the estimate can heavily lag behind the actual receive rate, especially when low.
+        //
+        // To mitigate this we override with the actual estimate plus some buffer if we aren't seeing packet loss.
+        if (
+          this.rateProbeState === RateProbeState.Probing &&
+          this.downlinkStats.usedBandwidthKbps > this.downlinkStats.bandwidthEstimateKbps &&
+          this.downlinkStats.packetsLost < VideoPriorityBasedPolicy.SPURIOUS_PACKET_LOST_THRESHOLD
+        ) {
+          this.logger.info(
+            `bwe: In probe state, overriding estimate ${this.downlinkStats.bandwidthEstimateKbps} with actual receive bitrate ${this.downlinkStats.usedBandwidthKbps}`
+          );
+          targetBitrate =
+            this.downlinkStats.usedBandwidthKbps +
+            VideoPriorityBasedPolicy.USED_BANDWIDTH_OVERRIDE_BUFFER_KBPS;
```

Comment:
How do you determine this buffer value?

Reply:
Totally arbitrary. Just has to be above what the average bitrates are. Since its still pretty small it shouldn't make it too aggressive or anything.